docs/docs/FAQ.mdx

```diff
@@ -40,6 +40,11 @@ If you still cannot login to the app, try the following:
 
 ## Assets
 
+### Does Immich change the file?
+
+No, Immich does not touch the original file under any circumstances,  
+all changes of the file are saved in the Side Car file and in the database.
```

Comment:
Is the side car file used only for photos that have an existing side car or will Immich create one if it doesn't exist?
```suggestion
all changes of the file are saved in the side car file and in the database.
```

Reply:
Immich creates one if none exists